int32 ActuatorID
---
float64 ACTUAL_CURRENT
float64 ACTUAL_VELOCITY
float64 ACTUAL_POSITION
float64 VOLTAGE
float64 CURRENT_SCALE
float64 VELOCITY_SCALE
float64 ACTUATOR_TEMPERATURE
float64 INVERTER_TEMPERATURE
bool    ACTUATOR_SWITCH
bool    ONLINE_STATUS
bool    INIT_STATE
